packages/mobx/src/types/observableannotation.ts

```diff
@@ -38,6 +46,62 @@ function extend_(
     )
 }
 
+function decorate_20223_(
+    this: Annotation,
+    desc,
+    context: ClassAccessorDecoratorContext | ClassFieldDecoratorContext
+) {
+    assert20223DecoratorType(context, ["accessor", "field"])
+
+    const ann = this
+    const { kind, name: key, addInitializer } = context
+
+    // Backwards/Legacy behavior
```

Comment:
Using `makeObservable` - ClassFieldDecorators (`@decorator x = 1`) aren't able to apply getters/setters, so it would instead add an annotation to be applied by a `makeObservable`, just like how decorators are presently implemented. More of a migration step than anything so that someone could turn on 2022.3 decorators, but not have to add `accessor` to everything if they're using current decorators.

Reply:
Ok, makes total sense now! Mind clarifying that in the comments?

Reply:
Done